birdhouse/deployment/deploy-data-raven-testdata-to-thredds.yml

```diff
@@ -1,12 +1,11 @@
 deploy:
-#- repo_url: git@github.com:Ouranosinc/raven.git
 - repo_url: https://github.com/Ouranosinc/raven-testdata
   # optional, default "origin/master"
-  # branch:
+  branch: origin/main
```

Comment:
Not directly related to this PR, but it would be good to update the default branch of https://github.com/Ouranosinc/raven-testdata because it doesn't match this one (`origin/master` is the current one).

Reply:
Absolutely. I was planning on doing that all at once. We'll be keeping `master` at least for a bit during the transition, but the new default will be `main`.

Reply:
This now done.